kratos/utilities/function_parser_utility.cpp

```diff
@@ -12,6 +12,9 @@
 //
 
 // System includes
+#ifdef KRATOS_MODERN_REGEX
+    #include <regex>
+#endif
```

Comment:
why is this needed?

Reply:
Because centos... (minimal version of gcc required for regex is 4.9, centos uses 4.8)